bump_versions.sh

```diff
@@ -19,23 +19,23 @@ CHART=${DIR}/Chart.yaml
 
 set -x
 
-OLD_VAULT_VERSION=$(helm show values  --jsonpath '{.image.tag}' ${DIR})
+OLD_VAULT_VERSION=$(helm show values --jsonpath '{.image.tag}' ${DIR})
 
 # replace supported vault version message in readme
-sed -i "s|$LINE $OLD_VAULT_VERSION|$LINE $NEW_VAULT_VERSION|g" ${README}
+sed -i '' "s|$LINE \`$OLD_VAULT_VERSION\`|$LINE \`$NEW_VAULT_VERSION\`|g" ${README}
```

Comment:
Is that mac vs linux change?

Reply:
Yes, it should still work on Linux, untested tho.